addon/helpers/type-of.js

```diff
@@ -1,15 +1,16 @@
+import {helper} from '@ember/component/helper'
```

Comment:
This is inconsistent with what you did in https://github.com/juwara0/ember-frost-core/blob/b94d42b0fa9d90fb89ee9fd2ec1ba05a6113a19d/addon/helpers/array.js as part of https://github.com/ciena-frost/ember-frost-core/pull/503

Reply:
This would be the updated way. The newer version of the code-mod is now doing it this way. I will open an issue to update `ember-frost-core`.